x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/indicator_match_rule_suppression.cy.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewThreatIndicatorRule } from '../../../../objects/rule';
+
+import {
+  DEFINITION_DETAILS,
+  SUPPRESS_FOR_DETAILS,
+  SUPPRESS_BY_DETAILS,
+  SUPPRESS_MISSING_FIELD,
+  DETAILS_TITLE,
+} from '../../../../screens/rule_details';
+
+import {
+  fillDefineIndicatorMatchRule,
+  fillAlertSuppressionFields,
+  selectIndicatorMatchType,
+  fillAboutRuleMinimumAndContinue,
+  createRuleWithoutEnabling,
+  skipScheduleRuleAction,
+  continueFromDefineStep,
+  selectAlertSuppressionPerInterval,
+  setAlertSuppressionDuration,
+  selectDoNotSuppressForMissingFields,
+} from '../../../../tasks/create_new_rule';
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { getDetails } from '../../../../tasks/rule_details';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+import { deleteAlertsAndRules } from '../../../../tasks/api_calls/common';
+
+const SUPPRESS_BY_FIELDS = ['myhash.mysha256', 'source.ip.keyword'];
+
+describe(
+  'Detection rules, Indicator Match, Alert Suppression',
+  {
+    tags: ['@ess', '@serverless'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'alertSuppressionForIndicatorMatchRuleEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    const rule = getNewThreatIndicatorRule();
+    beforeEach(() => {
+      cy.task('esArchiverLoad', { archiveName: 'threat_indicator' });
+      cy.task('esArchiverLoad', { archiveName: 'suspicious_source_event' });
+      deleteAlertsAndRules();
+      login();
+      visit(CREATE_RULE_URL);
+    });
+
+    it('creates rule with per rule execution suppression', () => {
+      selectIndicatorMatchType();
```

Comment:
NIT: Move `selectIndicatorMatchType();` and `fillDefineIndicatorMatchRule(rule);` beforeEach hook for both tests in this spec file.